src/Explorer/Graph/GraphExplorerComponent/GraphData.ts

```diff
@@ -413,13 +413,13 @@ export class GraphData<V extends GremlinVertex, E extends GremlinEdge> {
    * @param node
    * @param prop
    */
-  public static getNodePropValue(node: D3Node, prop: string): string | number | boolean {
+  public static getNodePropValue(node: D3Node, prop: string): undefined | string | number | boolean {
     if (node.hasOwnProperty(prop)) {
       return (node as any)[prop];
     }
 
     // This is DocDB specific
-    if (node.hasOwnProperty("properties") && node.properties.hasOwnProperty(prop)) {
+    if (node && node.properties && node.properties.prop && node.properties.prop[0]["value"]) {
```

Comment:
You can also use optional chaining for this which is a bit shorter, bit I have no strong preference one way or the other https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining